src/Sylius/Bundle/CoreBundle/spec/Checkout/CheckoutProcessScenarioSpec.php

```diff
@@ -51,7 +52,10 @@ function it_builds_checkout_process_with_proper_steps(ProcessBuilderInterface $b
         $builder->setDisplayRoute('sylius_checkout_display')->willReturn($builder)->shouldBeCalled();
         $builder->setForwardRoute('sylius_checkout_forward')->willReturn($builder)->shouldBeCalled();
 
-        $builder->setRedirect('sylius_homepage')->willReturn($builder)->shouldBeCalled();
+        $builder->setRedirect('sylius_account_order_show')->willReturn($builder)->shouldBeCalled();
+        $builder->setRedirectParams(array(
+            'number' => $order->getNumber()
+        ))->willReturn($builder)->shouldBeCalled();
```

Comment:
Without `shouldBeCalled` there is no assertion, so the spec will pass everytime, even when the `build` method will be empty. Why we want to have spec that doesn't validate anything?